polynote-frontend/polynote/ui/tags.ts

```diff
@@ -604,6 +605,11 @@ export function table(classes: string[], contentSpec: TableContentSpec): TableEl
                 rows.forEach(row => table.addRow(row, newBody));
             }
             return newBody;
+        },
+        clearAllRows() {
+            while (table.rows.length > 0) {
+                table.deleteRow(0);
```

Comment:
Will this leave the `tbody` elements around? I wonder if it'd be better to do something like `body.innerHTML = ""; table.innerHTML = ""; table.appendChild(body)` (or whatever variation of that gets us back to the table's initial state 🤔). 

If this is getting too complicated we can leave it for another PR.

Reply:
I'm confused by what "leave the `tbody` elements around" means - doesn't this code return us to the initial state? On opening the search modal for the first time, `table` has an empty `tbody` child element, and after clearing the rows, we get back to that.

Reply:
It's possible to have a table with multiple bodies (see `addBody`). I'm wondering what happens if we delete all the rows in such a table. Would you have a table with a bunch of empty `tbody` elements?

Reply:
Ah I gotcha! Yes, that's the current behavior, but I almost feel like that's preferable to deleting the `tbody` elements -  maybe whoever added them might still expect them to be there. What if we separate the functionality into two (😬 ) methods on `TableElement`, one that clears all the rows and one that returns the table to a "fresh" state with just one empty `tbody` element? Maybe that's too much complexity.